src/components/common/Table/styles.module.css

```diff
@@ -6,6 +6,16 @@
   max-width: 900px;
 }
 
+.gridEmptyRow {
+    display: grid;
+    grid-template-columns: 35% auto;
+    gap: var(--space-1);
+    justify-content: flex-start;
+    max-width: 900px;
+    margin-top: var(--space-1);
+    margin-bottom: var(--space-1);
+    border-top: 1px solid var(--color-border-light);
```

Comment:
Please run `yarn prettier`, this should be 2 spaces not 4.

Reply:
we'll we need this in our git hooks and also a check for this on CI, then. 

css was missing in my prettier conifg
![grafik](https://github.com/safe-global/safe-wallet-web/assets/693770/c4dff024-2d6e-490e-839a-dd27e09d2d4a)